src/citrine/resources/project.py

```diff
@@ -199,6 +206,49 @@ def make_private(self,
         })
         return True
 
+    def list_members(self) -> List[User]:
+        """
+        List all of the users in the current project.
+
+        Returns
+        -------
+        List[User]
+            The members of the current project
+
+        """
+        return self.session.checked_get(self._path() + "/users")["users"]
+
+    def set_member(self, user_uid, role: str, actions: List[str]) -> bool:
+        """
+        Add a User to a Project with the given 'role' and 'actions'.
+
+        Returns
+        -------
+        bool
+            Returns True if member successfully added
```

Comment:
To me this isn't meaningful when an exception is thrown rather `False` being returned when the member is not added successfully. Might as well not have a return value.

Reply:
We could also return a `Response` object

Reply:
Sorry, I was looking at this out of context of all the changes already having been approved.